frontend/app/routes/settings/change-password.tsx

```diff
@@ -0,0 +1,180 @@
+
+  throw redirect("/settings/account");
+}
+
+export default function UserSettingsPage({}: Route.ComponentProps) {
+  return (
+    <section className="flex flex-col gap-4">
+      <div className="flex items-center gap-4">
+        <h2 className="text-2xl">Change Password</h2>
+      </div>
+      <Separator />
+
+      <p className="text-gray">
+        Update your account password. Make sure to use a strong password
+      </p>
+
+      <ChangePassword />
+    </section>
+  );
+}
+
+function ChangePassword() {
+  const navigation = useNavigation();
+  const fetcher = useFetcher<typeof clientAction>();
+  const isPending =
+    navigation.state === "loading" || navigation.state === "submitting";
+  const errors = getFormErrorsFromResponseData(fetcher.data?.errors);
+
+  const formRef = React.useRef<React.ComponentRef<"form">>(null);
+
+  return (
+    <fetcher.Form
+      method="POST"
+      ref={formRef}
+      className="space-y-6 animate-in fade-in duration-300 max-w-lg"
+    >
+      {errors.non_field_errors && (
+        <Alert variant="destructive">
+          <AlertCircle className="h-4 w-4" />
+          <AlertTitle>Error</AlertTitle>
+          <AlertDescription>{errors.non_field_errors}</AlertDescription>
+        </Alert>
+      )}
+      <FieldSet
+        name="current_password"
+        required
+        errors={errors.current_password}
+        className="space-y-2"
+      >
+        <FieldSetLabel className="block">Current Password</FieldSetLabel>
+        <FieldSetPasswordToggleInput
+          placeholder="Enter your current password"
+          label="Current Password"
+        />
+      </FieldSet>
+      <Separator />
+      <div className="space-y-1 text-muted-foreground">
+        <h3 className="font-medium text-sm">Hints for a good password</h3>
+        <ul className="list-disc list-inside text-xs">
+          <li>Use a mix of uppercase, lowercase, numbers, and symbols.</li>
+          <li>Avoid using common passwords.</li>
+          <li>Make it long and hard to guess.</li>
+        </ul>
+      </div>
+      <FieldSet
+        name="new_password"
+        required
+        errors={errors.new_password}
+        className="space-y-2"
+      >
+        <FieldSetLabel className="block">New Password</FieldSetLabel>
+        <FieldSetPasswordToggleInput
+          placeholder="Enter your new password"
+          label="New Password"
+        />
+      </FieldSet>
+      <FieldSet
+        name="confirm_password"
+        required
+        errors={errors.confirm_password}
+        className="space-y-2"
+      >
+        <FieldSetLabel className="block">Confirm New Password</FieldSetLabel>
+        <FieldSetPasswordToggleInput
+          placeholder="Confirm your new password"
+          label="Confirm Password"
+        />
+      </FieldSet>
+      <SubmitButton
+        isPending={isPending}
+        name="intent"
+        value="change_password"
+        variant="secondary"
```

Comment:
```suggestion
        variant="default"
```
This is an important form, so it deserves the primary variant (which is the default,), also no need for intent